kod/object/item/passitem/numbitem/ammo.kod

```diff
@@ -81,8 +86,17 @@ messages:
 
    GetProjectileLightFlags()
    {
-      % No lighting information.
-      return LIGHT_FLAG_NONE;
+      return viProjectileLightFlags;
+   }
+
+   GetProjectileLightIntensity()
+   {
+      return viProjectileLightIntensity;
+   }
+
+   GetProjectileLightColor()
```

Comment:
I'm not sure these extra functions are needed.  Looks like your new code calls only GetProjectileLightFlags?